oidc-controller/api/main.py

```diff
@@ -92,15 +92,22 @@ async def logging_middleware(request: Request, call_next) -> Response:
     finally:
         process_time = time.time() - start_time
         # If we have a response object, log the details
-        if 'response' in locals():
-            logger.info("processed a request", status_code=response.status_code, process_time=process_time)
+        if "response" in locals():
```

Comment:
You don't have to change this but I think in the future I would handle this by declaring response as `None` and then checking if `response == None` instead of checking the variable string name is in the locals array.